docs/markdown/Reference-tables.md

```diff
@@ -36,6 +36,10 @@ These are return values of the `get_id` (Compiler family) and
 | valac     | Vala compiler                    |                 |
 | xc16      | Microchip XC16 C compiler        |                 |
 | cython    | The Cython compiler              |                 |
+| nasm      | The NASM compiler (Since 0.64.0) |                 |
+| yasm      | The YASM compiler (Since 0.64.0) |                 |
+| ml        | Microsoft Macro Assembler for x86 and x86_64 (Since 0.64.0) | msvc |
+| armasm    | Microsoft Macro Assembler for ARM and AARCH64 (Since 0.64.0) | |
```

Comment:
`armasm` is missing the `msvc` entry in the second column?

Reply:
I don't think so, because ml has msvc-like CLI but armasm has gnu-like...